network-api/networkapi/templates/pages/buyersguide/category_page.html

```diff
@@ -27,6 +27,20 @@ <h2>({{ category.name }}) Primary related articles (First 3 of list of 6)</h2>
           <div>
             <a href="{% pageurl page.get_editorial_content_index %}">All articles</a>
           </div>
+          {% with cta=category.call_to_action %}
+            {% if cta %}
+              <div>
+                <h2> Call to Action: </h2>
+                <p> <b>Title:</b> {{ cta.title }} </p>
+                <p> <b>Sticker Image:</b> {{ cta.sticker_image }} </p>
+                <p> <b> Content: </b> </p>
+                {{ cta.content|richtext }}
+                <p> <b>Link Label: </b> {{cta.link_label }} </p>
+                <p> <b> Link URL (if applicable): </b> {{cta.link_target_url }} </p>
+                <p> <b> Link Page (if applicable): </b> {{cta.link_target_page }} </p>
```

Comment:
**Suggestion** It's usually best to keep this logic (URL or page) out of the template and add a method to the object that returns the correct URL.

Reply:
Updated!